modules/installation/pages/ports.adoc

```diff
@@ -72,8 +72,6 @@ In most cases, you will not need to adjust these ports.
 | Port number | Notes
```

Comment:
```suggestion
| Port Number | Notes
```
Drive by edit.  I'd capitalize column titles (title style).  But we can do this later.